lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -8375,6 +8400,8 @@ class DecomposeComplexOpsPass
     addPatternIfTargetOpIsIllegal<DecomposeAtenReshapeAsOp>(patterns);
     addPatternIfTargetOpIsIllegal<DecomposeAtenTriuOp>(patterns);
     addPatternIfTargetOpIsIllegal<DecomposeAtenLinalgNormOp>(patterns);
+    addPatternIfTargetOpIsIllegal<
+        DecomposeAtenFakeQuantizePerTensorAffineCachemaskOp>(patterns);
```

Comment:
Actually, on a second glance, does the op need to be made illegal in [/lib/Dialect/Torch/Transforms/LowerToBackendContract.cpp](https://github.com/llvm/torch-mlir/blob/694210f429e44977c35fb5d8890f63b89aaad0ed/lib/Dialect/Torch/Transforms/LowerToBackendContract.cpp#L369)?

Reply:
done